styles.css

```diff
@@ -147,7 +152,24 @@ main {
   margin-left: 1%;
   flex-direction: column;
   box-shadow:  5px 5px 10px #b3b3b3,
-             -5px -5px 10px #ffffff;
+  -5px -5px 10px #ffffff;
+}
+
+.card-div {
+  margin-left: 10px;
+  margin-right: 10px;
```

Comment:
You can short hand this

`margin: 0 10px 0 10px;`

Reply:
Done!